win32/MainWindow.xaml.cs

```diff
@@ -674,8 +674,8 @@ private void CsvExport_SingleCurveInit()
             {
                 new ModelArgs()
                 {
-                    h_1__user_units = Tools.ConvertToMeters(inputControl.h_1),
-                    h_2__user_units = Tools.ConvertToMeters(inputControl.h_2),
+                    h_1__user_units = inputControl.h_1,
```

Comment:
Is there ever a case where this might need to be converted to meters, or was the conversion call redundant?

Reply:
no, the model only deals with meters, white the GUI keeps track of UI values in user_units